src/Compilers/VisualBasic/Portable/Syntax/VisualBasicSyntaxTree.vb

```diff
@@ -154,28 +154,62 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
         Friend Shared ReadOnly DummyReference As SyntaxReference = Dummy.GetReference(Dummy.GetRoot())
 
+        ''' <summary>
+        ''' Creates a new syntax tree from a syntax node.
+        ''' </summary>
+        Public Shared Function Create(root As VisualBasicSyntaxNode,
+                                      options As VisualBasicParseOptions,
+                                      path As String,
+                                      encoding As Encoding) As SyntaxTree
+            Return CreateImpl(root, diagnosticOptions:=Nothing, options, path, encoding, SourceHashAlgorithm.Sha1)
+        End Function
+
         ''' <summary>
         ''' Creates a new syntax tree from a syntax node.
         ''' </summary>
         ''' <param name="diagnosticOptions">An obsolete parameter. Diagnostic options should now be passed with <see cref="CompilationOptions.SyntaxTreeOptionsProvider"/></param>
+        <EditorBrowsable(EditorBrowsableState.Never)>
+        <Obsolete("The diagnosticOptions and isGeneratedCode parameters are obsolete due to performance problems, if you are using them use CompilationOptions.SyntaxTreeOptionsProvider instead", False)>
```

Comment:
There is no `isGeneratedCode` parameter here. Is this a new Obsoletion?

Reply:
I copied the attribute from C#. Not sure why VB did not have isGenerated. Will remove it from the text.

Reply:
The same API in C# was already obsolete, so making this obsolete brings them on par.